examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/projects_smaller.csv

```diff
@@ -0,0 +1,1001 @@
+﻿1,https://api.github.com/repos/tosch/ruote-kit,1,ruote-kit,RESTful wrapper for ruote workflow engine,Ruby,12/8/2009 10:17,2,0,11/5/2015 1:15

```

Comment:
Why are we including this?  Can't users isn't there already a small sample projects file within the dataset that users can download ?  Also is uploading their dataset to this repo violating their license in any way (same question regarding the yelp dataset that was uploaded as well)? http://ghtorrent.org/faq.html

Reply:
I don't believe we've uploaded a subset of this dataset to the repo before/that a smaller subset is available anywhere on GHTorrent without the user having to download the many Gig dataset and then take a subset of it? A subset of the projects data was [included in this earlier PR](https://github.com/dotnet/spark/blob/6037a3075f1fc4839c13069a91977f6b7689aa66/scenarios/datasets/projects.csv) (but was never merged).

I believe the Yelp dataset should be fine since we listed that it came from the UCI dataset: https://archive.ics.uci.edu/ml/datasets/Sentiment+Labelled+Sentences.

As for GHTorrent, "For non-commercial uses (including, but not limited to, educational, research or personal uses), the dataset is distributed under the CC-BY-SA license." This license (found at https://creativecommons.org/licenses/by-sa/4.0/) says we're free to share and adapt. So I believe it follows their license, if you are also interpreting it the same way?

Reply:
I had also gotten a request to upload a smaller/more user-friendly version of the dataset for docs purposes (so that we can link users to this dataset directly from the documentation rather than potentially intimidating them by telling them to go download a huge file).